docs/api-reference/retention-rules-api.md

```diff
@@ -25,45 +25,464 @@ sidebar_label: Retention rules
+
+<code class="getAPI">GET</code> `/druid/coordinator/v1/rules`
+
+### Responses
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--200 SUCCESS-->
+<br/>
+*Successfully retrieved retention rules* 
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+---
+
+### Sample request
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--cURL-->
+```shell
+curl "http://ROUTER_IP:ROUTER_PORT/druid/coordinator/v1/rules"
+```
+<!--HTTP-->
+```HTTP
+GET /druid/coordinator/v1/rules HTTP/1.1
+Host: http://ROUTER_IP:ROUTER_PORT
+```
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+### Sample response
+
+<details>
+  <summary>Click to show sample response</summary>
+
+  ```json
+{
+    "_default": [
+        {
+            "tieredReplicants": {
+                "_default_tier": 2
+            },
+            "type": "loadForever"
+        }
+    ],
+    "social_media": [
+        {
+            "interval": "2023-01-01T00:00:00.000Z/2023-02-01T00:00:00.000Z",
+            "type": "dropByInterval"
+        }
+    ],
+    "wikipedia_api": [],
+}
+  ```
+</details>
+
+## Get an array of retention rules for a datasource
+
+Retrieves an array of rule objects for a single datasource. If there are no retention rules, it returns an empty array. 
+
+Note that this endpoint returns an `HTTP 200 Success` code message even if the `datasource` does not exist.
+
+### URL
+
+<code class="getAPI">GET</code> `/druid/coordinator/v1/rules/{dataSource}`
+
+### Query parameters
+* `full` (optional)
+  * Include the default retention rule for the datasource in the response.
+
+### Responses
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--200 SUCCESS-->
+<br/>
+*Successfully retrieved retention rules* 
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+---
+
+### Sample request
+
+The following example retrieves the custom retention rules and default retention rules for datasource with specified ID `social_media`.
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--cURL-->
+```shell
+curl "http://ROUTER_IP:ROUTER_PORT/druid/coordinator/v1/rules/social_media?full=null"
+```
+<!--HTTP-->
+```HTTP
+GET /druid/coordinator/v1/rules/social_media?full=null HTTP/1.1
+Host: http://ROUTER_IP:ROUTER_PORT
+```
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+### Sample response
+
+<details>
+  <summary>Click to show sample response</summary>
+
+  ```json
+[
+    {
+        "interval": "2020-01-01T00:00:00.000Z/2022-02-01T00:00:00.000Z",
+        "type": "dropByInterval"
+    },
+    {
+        "interval": "2010-01-01T00:00:00.000Z/2020-01-01T00:00:00.000Z",
+        "tieredReplicants": {
+            "_default_tier": 2
+        },
+        "type": "loadByInterval"
+    },
+    {
+        "tieredReplicants": {
+            "_default_tier": 2
+        },
+        "type": "loadForever"
+    }
+]
+  ```
+</details>
+
+## Get audit history for all datasources
+
+Retrieves the audit history of rules for all datasources over an interval of time. The default value of `interval` can be specified by setting `druid.audit.manager.auditHistoryMillis` (1 week if not configured) in Coordinator `runtime.properties`.
```

Comment:
Id'd rephrase this since the naming/file structure is a little unintuitive for Druid since all the runtime properties are just called `runtime.properties` regardless of service besides common and you have to check the directory

- in the runtime properties for the Coordinator 
- in the `runtime.properties` file for the Coordinator